packages/browser-wallet/src/popup/pages/IdentityIssuance/IdentityIssuanceEnd.tsx

```diff
@@ -25,18 +25,14 @@ interface Location {
     };
 }
 
-interface Props {
-    onFinish: () => void;
-}
-
 // TODO start checking status on identity
 
-export default function IdentityIssuanceEnd({ onFinish }: Props) {
+export default function IdentityIssuanceEnd() {
```

Comment:
This component should have some padding around the text that matches what we use elsewhere. Currently it touches the boundaries of the window.